posts/inside-rust/2024-08-12-announcing-metrics-initiative.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: post
+title: "Announcing the Metrics Initiative"
+author: Jane Losare-Lusby and Esteban Kuber
+team: The Rust Compiler Team <https://www.rust-lang.org/governance/teams/compiler>
+---
+
+We ([Jane](https://github.com/yaahc/) and [Esteban](https://github.com/estebank)) are excited to announce the start of the Metrics initiative.
+
+We're envisioning three use cases for metrics within rustc.
+
+* Supporting feature development
+    * e.g., answering specific questions such as when the old and new trait solvers diverge or helping identify and resolve bugs before impacting users
+* Guiding improvements to User Experience,
+    * e.g., knowing which compiler errors are causing the most confusion or are hit the most frequently, focusing on improving those first, and verifying that the improvements help
+* Improving perf feedback loops and insight,
+    * e.g., helping identify pathological edge cases, similar to work @nnethercote has done manually in the past
```

Comment:
```suggestion
    * e.g., helping identify pathological edge cases, similar to [work](https://nnethercote.github.io/2024/03/06/how-to-speed-up-the-rust-compiler-in-march-2024.html) @nnethercote has done manually in the past
```
We can add a link to Nick's blog post, not everyone might be familiar with this.

Reply:
changing to the blog post that has the relevant analysis of the top crates on crates.io but otherwise +1

```suggestion
    * e.g., helping identify pathological edge cases, similar to [work](https://nnethercote.github.io/2022/02/25/how-to-speed-up-the-rust-compiler-in-2022.html) @nnethercote has done manually in the past
```